specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1738,6 +2169,13 @@
       "maxLength": 256,
       "pattern": "^[^*#&+:<>?]+$",
       "x-ms-parameter-location": "method"
+    },
+    "IfMatch": {
```

Comment:
This is generic enough to go in [https://github.com/darrelmiller/azure-rest-api-specs/blob/current/specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimanagement.json#L520](https://github.com/darrelmiller/azure-rest-api-specs/blob/current/specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimanagement.json#L520)

Reply:
Will do.